src/Services/SearchService.php

```diff
@@ -364,4 +367,31 @@ private function getVariationIdForRedirect(string $query, array $documents)
 
         return $mainVariationId;
     }
+
+    /**
+     * @see \IO\Extensions\Filters\URLFilter::buildItemURL (source)
+     */
+    private function buildItemURL($itemData, $withVariationId = true): string
```

Comment:
Passing the variation id directly would make the flag unnecessary, right?

Reply:
I want to stick with the plenty implementation, so it can be updated easily in the future if it changes.